### DISCUSSION TITLE
CI/CD pipeline generator

### DISCUSSION THREAD

**Discussion by matt-psaltis** [cake-build/cake#3961] [Ideas]
Not sure if this is hiding somewhere already for cake but I'd love support for this Nuke feature in Cake!

Example of feature here: https://nuke.build/docs/cicd/azure-pipelines/#configuration-generation

Would be happy to contribute if this was to get support and a green light :D

**Comment by nils-a** [comment]
Sorry for not answering earlier.

I think the idea is brilliant!

However, I think it would be best to start this as a separate project - this will make it easier to get an initial version out and tested.
